better-font-awesome.php

```diff
@@ -128,21 +128,21 @@ class Better_Font_Awesome_Plugin {
      *
      * @return  Better_Font_Awesome  The BFA object.
      */
-    public static function get_instance( $args = '' ) {
-        static $instance = null;
-        if ( null === $instance ) {
-            $instance = new static( $args );
-        }
+	public static function get_instance( $args = '' ) {
 
-        return $instance;
-    }
+		if ( ! self::$instance ) {
+			self::$instance = new static( $args );
```

Comment:
Can't think of a case where you'd require pass in of args to the instance getter and maintain a strict singleton, just put those args into the instance getter. 

I've never really used strict singletons because more often than not it makes sense to have access to the class with custom args.

In this case, I think you're good to use a strict singleton but remove instance getter args because  1. They aren't used for anything 2. If they were used for something, you wouldn't want a strict singleton